### ISSUE TITLE
Downloading Problem

### ISSUE THREAD

**Issue by Iwastaken25** [PibePlayer/MultiMC5-Cracked#13]
### Role

I Speedrun Minecraft 1.16.1

### Suggestion

I Cant Download Multimc Cracked it Says that the file does not exist or expired

### Benefit

So That I can Download MultiMC Again

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by ghost** [comment]
If you don't feel like wasting your time installing then you can use any other cracked minecraft launcher. Be wary though because some can contain viruses

**Comment by jroprohehe** [comment]
Same

**Comment by xknat** [comment]
i hope @PibePlayer will pick this up

**Comment by PibePlayer** [comment]
Yep, checking it now, the Qt installation on Windows (to compile) seems to be stuck.
I'll debug it after work

**Comment by pesaksintondji** [comment]
https://nightly.link/UltimMC/Launcher/workflows/main/develop
I find this link today by googling "mmc cracked", The file work fine for me.Hope it help you.
